app/cluster/state/etcd.go

```diff
@@ -30,8 +31,8 @@ var (
 )
 
 const (
-	modeRequestKeyPattern        = `/%s/server/%s/mode`       // /<releaseName>/server/<serverIndex>/mode
-	workspacesRequestsKeyPattern = `/%s/server/%s/workspaces` // /<releaseName>/server/<serverIndex>/workspaces
+	modeRequestKeyPattern        = `/%s/server/%s/mode`          // /<releaseName>/server/<serverIndex>/mode
+	workspacesRequestsKeyPattern = `/%s/server/%s/%s/workspaces` // /<releaseName>/server/<serverIndex>/<app_type>/workspaces
```

Comment:
Can you please explain why the APP_TYPE is needed now? Thanks.

Reply:
@cisse21